src/application/services/property.service.ts

```diff
@@ -81,6 +86,30 @@ class PropertyService implements IPropertyService {
     }
     return owner;
   }
+
+  public async getMyProperties(
+    ownerEmail: string,
+  ): Promise<ResponseDtos.PropertiesDto> {
+    const owner = await this.getOwnerFromEmail(ownerEmail);
+    const properties = await this.propertyRepository.findByOwnerId(owner.id);
+    return { properties: properties.map(PropertyFactory.toDto) };
+  }
+
+  public async delete(
+    propertyId: string,
+    ownerEmail: string,
+  ): Promise<ResponseDtos.PropertyDto> {
+    const owner = await this.getOwnerFromEmail(ownerEmail);
+    const property = await this.propertyRepository.findById(propertyId);
+    if (!property) {
+      throw new NotFoundException('Property does not exist');
+    }
+    if (owner.id !== property.ownerId) {
+      throw new DomainException('Property does not belong to the user');
+    }
+    await this.propertyRepository.deleteById(propertyId);
+    return PropertyFactory.toDto(property);
```

Comment:
Why are we returning a deleted property? We should not return anything here

Reply:
Es por si quieres mostrar alguna informacion de la propery luego de eliminarla tipo: La propiedad en Calle falsa 123 depto69 ha sido eliminada. qcy igual lo quito